extras/cloudsql-multicluster/tls-for-mci.md

```diff
@@ -0,0 +1,117 @@
+# Enable TLS support on the Multi Cluster Ingress
+---
+
+In this section we will enable **TLS** support _(with
+**HTTP to HTTPS** redirection)_ on the **Multi Cluster Ingress** resource that was
+created in the [Multi Cluster Bank of Anthos with Cloud SQL](README.md) guide.
+
+## Prerequisites
+
+- Bank of Anthos deployed in multiple clusters with access to Cloud SQL as shown
+  in the [previous guide](README.md).
+- This guide assumes that all the environment variables and **kubectx**(s) -
```

Comment:
Praise: I'm glad you added this note about keeping the environment variable in tack.
I accidentally used a separate terminal tab at first. 😅